src/App.tsx

```diff
@@ -1,15 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [search, setSearch] = useState('');
+
+  const searchFilter = (movie: Movie) => {
+    const { title, description } = movie;
+
+    return title.toLowerCase().includes(search.toLowerCase())
+      || description.toLowerCase().includes(search.toLowerCase());
+  };
+
```

Comment:
You didnt handle cases with additional white spaces around the search query. That is why your tests were failed

Reply:
Also, you have used toLowerCase method two times for the one variable. I think, it will be better if you create additional variable to reuse it